### PR TITLE
upgraded scala to 2.13.10

### PR DESCRIPTION
**Pull request by mauro-fanton**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [ ] `./precheck` was executed (Integration/Component/Unit tests)
 - [ ] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.

### CODE REVIEW

File: test/uk/gov/hmrc/plasticpackagingtaxreturns/connectors/FinancialDataConnectorISpec.scala [hunk 1]
```diff
@@ -16,36 +16,39 @@
...
 package uk.gov.hmrc.plasticpackagingtaxreturns.connectors
 
-import com.codahale.metrics.SharedMetricRegistries
-import com.github.tomakehurst.wiremock.client.WireMock._
-import org.mockito.Mockito
-import org.mockito.Mockito.when
+import com.codahale.metrics.Timer
+import com.kenshoo.play.metrics.Metrics
+import org.mockito.ArgumentCaptor
+import org.mockito.ArgumentMatchers.{eq => meq}
```
Comment (lmcewanhmrc) [inline test/uk/gov/hmrc/plasticpackagingtaxreturns/connectors/FinancialDataConnectorISpec.scala:22]: That's the java matchers

File: test/uk/gov/hmrc/plasticpackagingtaxreturns/connectors/FinancialDataConnectorISpec.scala [hunk 2]
```diff
@@ -16,36 +16,39 @@
...
+import org.mockito.ArgumentMatchersSugar.any
+import org.mockito.Mockito.{mock => _, _}
```
Comment (lmcewanhmrc) [inline test/uk/gov/hmrc/plasticpackagingtaxreturns/connectors/FinancialDataConnectorISpec.scala:24]: Java mockito

File: test/uk/gov/hmrc/plasticpackagingtaxreturns/connectors/FinancialDataConnectorISpec.scala [hunk 3]
```diff
@@ -16,36 +16,39 @@
...
 import org.scalatest.BeforeAndAfterEach
 import org.scalatest.Inspectors.forAll
-import org.scalatest.concurrent.Eventually.eventually
-import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.time.{Seconds, Span}
-import play.api.Application
-import play.api.http.Status
-import play.api.inject.bind
-import play.api.inject.guice.GuiceApplicationBuilder
+import org.scalatestplus.mockito.MockitoSugar._
```
Comment (lmcewanhmrc) [inline test/uk/gov/hmrc/plasticpackagingtaxreturns/connectors/FinancialDataConnectorISpec.scala:28]: Old version of Scala mockito :D

File: test/uk/gov/hmrc/plasticpackagingtaxreturns/connectors/FinancialDataConnectorISpec.scala [hunk 4]
```diff
@@ -16,36 +16,39 @@
...
-import com.github.tomakehurst.wiremock.client.WireMock._
-import org.mockito.Mockito
-import org.mockito.Mockito.when
+import com.codahale.metrics.Timer
+import com.kenshoo.play.metrics.Metrics
+import org.mockito.ArgumentCaptor
+import org.mockito.ArgumentMatchers.{eq => meq}
+import org.mockito.ArgumentMatchersSugar.any
+import org.mockito.Mockito.{mock => _, _}
+import org.mockito.MockitoSugar.{reset, verify, when}
```
Comment (lmcewanhmrc) [inline test/uk/gov/hmrc/plasticpackagingtaxreturns/connectors/FinancialDataConnectorISpec.scala:25]: New version of Scala mockito

File: test/uk/gov/hmrc/plasticpackagingtaxreturns/connectors/FinancialDataConnectorISpec.scala [hunk 5]
```diff
@@ -16,36 +16,39 @@
...
 
-import com.codahale.metrics.SharedMetricRegistries
-import com.github.tomakehurst.wiremock.client.WireMock._
-import org.mockito.Mockito
-import org.mockito.Mockito.when
+import com.codahale.metrics.Timer
+import com.kenshoo.play.metrics.Metrics
+import org.mockito.ArgumentCaptor
+import org.mockito.ArgumentMatchers.{eq => meq}
+import org.mockito.ArgumentMatchersSugar.any
```
Comment (lmcewanhmrc) [inline test/uk/gov/hmrc/plasticpackagingtaxreturns/connectors/FinancialDataConnectorISpec.scala:23]: These are the Scala matchers 👍

File: test/uk/gov/hmrc/plasticpackagingtaxreturns/controllers/models/returns/ReturnValuesSpec.scala [hunk 6]
```diff
@@ -34,7 +34,7 @@ class ReturnValuesSpec extends PlaySpec {
 
       result mustBe Some(NewReturnValues(
         periodKey = "21C4",
-        periodEndDate = LocalDate.now,
+        periodEndDate = LocalDate.of(2022, 7, 1),
```
Comment (lmcewanhmrc) [inline test/uk/gov/hmrc/plasticpackagingtaxreturns/controllers/models/returns/ReturnValuesSpec.scala:37]: nice 👍

**Comment by mauro-fanton** [comment]
Rebased from the wrong branch (partridge) instead of main

**Pull request closed without merging**